src/configuration/Configuration.ts

```diff
@@ -286,6 +324,138 @@ export class Configuration implements IConfigurationSettings {
     this._dataExclusions = Utils.addRange<string>(this._dataExclusions, ...exclusions);
   }
 
+  /**
+   * Gets a value indicating whether to include private information about the local machine.
+   * @returns {boolean}
+   */
+  public get includePrivateInformation(): boolean {
+    return this._includePrivateInformation;
+  }
+
+  /**
+   * Sets a value indicating whether to include private information about the local machine
+   * @param value
+   */
+  public set includePrivateInformation(value: boolean) {
+    const val = value || false;
+    this._includePrivateInformation = val;
+    this.includeUserName = val;
+    this._includeMachineName = val;
+    this.includeIpAddress = val;
+    this.includeCookies = val;
+    this.includePostData = val;
+    this.includeQueryString = val;
+    this.changed();
+  }
+
+  /**
+   * Gets a value indicating whether to include User Name.
+   * @returns {boolean}
+   */
+  public get includeUserName(): boolean {
+    return this._includeUserName;
+  }
+
+  /**
+   * Sets a value indicating whether to include User Name.
+   * @param value
+   */
+  public set includeUserName(value: boolean) {
```

Comment:
currently nothing using this, but may be useful on node? These were synced from .net client